std/path.d

```diff
@@ -3397,7 +3397,7 @@ do
     else
     {
         C[] pattmp;
-        foreach (ref pi; 0 .. pattern.length)
+        for (auto pi = 0; pi < pattern.length; pi++)
```

Comment:
`auto pi = 0` will give `pi` the type `int`, not `size_t` as it was before.

Reply:
Oops, fixed.